src/main/java/org/broadinstitute/hellbender/engine/FeatureDataSource.java

```diff
@@ -374,7 +374,7 @@ final void printCacheStats() {
             final boolean requireIndex = false;
 
             // Only apply the wrappers if the feature input is on Google Cloud Storage
-            if (BucketUtils.isCloudStorageUrl(featureInput)) {
+            if (BucketUtils.isEligibleForPrefetching(featureInput)) {
```

Comment:
The comment "Only apply the wrappers if the feature input is on Google Cloud Storage" needs updating.

Reply:
done